py3status/modules/xrandr.py

```diff
@@ -333,9 +333,14 @@ def _apply(self, force=False):
             self.active_mode = mode
         self.py3.log('command "{}" exit code {}'.format(cmd, code))
 
+        self._apply_background()
```

Comment:
Hmm, let's just skip this function and go with this instead. 
```diff
+        if self.command:
+            self.py3.command_run(self.command)
```

Reply:
It's simple enough indeed.